itest/universerpc_harness.go

```diff
@@ -0,0 +1,52 @@
+package itest
+
+import (
+	"github.com/lightninglabs/taproot-assets/proof"
+	"github.com/lightningnetwork/lnd/lntest/node"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+// UniverseRPCHarness is an integration testing harness for the universe tap
+// service.
+type UniverseRPCHarness struct {
+	// service is the instance of the universe tap service.
+	service *tapdHarness
+
+	// ListenAddr is the address that the service is listening on.
+	ListenAddr string
+}
+
+// NewUniverseRPCHarness creates a new test harness for a universe tap service.
+func NewUniverseRPCHarness(t *testing.T, ht *harnessTest,
```

Comment:
It's customary to return a pointer to a struct. That way in `itest/test_harness.go` you can simply do:

```
		proofCourier = NewUniverseRPCHarness(t, ht, lndHarness.Bob)
```